dome9/resource_dome9_service_account.go

```diff
@@ -0,0 +1,119 @@
+package dome9
+
+import (
+	"github.com/dome9/dome9-sdk-go/dome9/client"
+	"github.com/dome9/dome9-sdk-go/services/serviceaccounts"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"log"
+)
+
+func resourceServiceAccount() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceServiceAccountCreate,
+		Read:   resourceServiceAccountRead,
+		Update: resourceServiceAccountUpdate,
+		Delete: resourceServiceAccountDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
```

Comment:
what does Importer means? @itzikYeret @paveli-d9

Reply:
I don't know, I copied this from another resource